winmake.ps1

```diff
@@ -72,7 +72,8 @@ function Win-SSHProxy {
 
     New-Item -ItemType Directory -Force -Path "./bin/windows"
     if (-Not $Version) {
-        $Version = "v0.8.4"
+        $match = Select-String -Path ".\go.mod" -Pattern "github.com/containers/gvisor-tap-vsock\s+(v[\d\.]+)"
```

Comment:
```suggestion
        $match = Select-String -Path "$PSScriptRoot\go.mod" -Pattern "github.com/containers/gvisor-tap-vsock\s+(v[\d\.]+)"
```

In a lot of places we (incorrectly) assume that `winmake.ps1` is executed from the root directory, but for new code we should avoid that.

Reply:
I have add you suggestion